### ISSUE TITLE
Media thumbnails not working

### ISSUE THREAD

**Issue by inghamn** [City-of-Bloomington/uReport#226]
It  might be a file permissions problem, but thumbnails are all coming back as 404 again.

We should make sure we get a Skidder error whenever the thumbnail generation code errors out.

**Comment by inghamn** [comment]
This problem comes from the multi-home hosting we're doing.  Each site's media directory needs to be aliased in Apache, which is overriding the  route to the Media controller's actions.

We should choose a different alias for hosting the images, so there's no conflict with the Media controller.  In our media-manager application, we host the images at /m.  That would probably not conflict with anything we're doing here.

**Comment by inghamn** [comment]
I ended up leaving the alias as /media.  I just had to add the same rewrite rules into the /media alias.  Here's an example config

```apache
Alias /crm/media "${PROJECTS}/uReport/uReport/crm/data/media"
<Directory       "${PROJECTS}/uReport/uReport/crm/data/media">
    Options FollowSymLinks
    AllowOverride None
    Require all granted

    RewriteBase /crm
    RewriteCond %{REQUEST_FILENAME} !-d
    RewriteCond %{REQUEST_FILENAME} !-f
    RewriteRule .? /crm/index.php [NC,L]
</Directory>

Alias /crm "${PROJECTS}/uReport/uReport/crm/public"
<Directory "${PROJECTS}/uReport/uReport/crm/public">
    Options FollowSymLinks
    AllowOverride None
    Require all granted

    RewriteBase /crm
    RewriteCond %{REQUEST_FILENAME} !-d
    RewriteCond %{REQUEST_FILENAME} !-f
    RewriteRule .? /crm/index.php [NC,L]
</Directory>
```

**Comment by rbdietz** [comment]
Make sure this is reflected in install instructions.  Thanks.

On Tue, May 24, 2016 at 10:36 AM, Cliff Ingham <notifications@github.com>
wrote:

> Closed #226 <https://github.com/City-of-Bloomington/uReport/issues/226>.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/City-of-Bloomington/uReport/issues/226#event-670420423>
>



-- 
~~~~~~~~~~~~~~~~~~~~~~~~~
Rick Dietz, Director
Information & Technology Services Dept.
City of Bloomington, Indiana
812.349.3485 (office) 812.340.7958 (cell)
dietzr@bloomington.in.gov
